docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -6,66 +6,175 @@ aliases = ["/docs/grafana/latest/enterprise/license-restrictions"]
 
-- When the number of maximum active viewers or editor/admins is reached, only those currently active users can sign in. New users or non-active users cannot sign in.
-- A license limit banner will appear to admins when Grafana reaches its active user limit. Editor/admins and viewers will not see the banner.
-- To see how many active users you have in each licensed role (Viewer or Editor/Admin), refer to the Licensing page in the Server Admin section of Grafana, which is located at `[your-grafana-url.com]/admin/licensing`. Please note that _licensed_ roles can differ from the Active Viewer/Editor/Admin counts on the /admin/stats page in Grafana. This is because the Stats page only counts a user's assigned organization role and does not account for dashboard and folder permissions.
-- Restrictions are applied separately for viewers and editor/admins. If a Grafana instance reaches its limit of active viewers but not its limit of active editor/admins, new editors and admins will still be able to sign in.
-- You can change a user's licensed role by updating their permissions in Grafana (their role or their dashboard/folder permissions). Their new role will go into effect the next time that user signs in.
-- Active user counts are not affected by the number of dashboards, folders, or organizations a user can edit or admin. An active user who can edit many dashboards or folders in many different orgs is still counted as a single editor.
+1. Hover your mouse over **Server Admin** (the shield icon), and click **Stats & licensing**.
 
-### Concurrent sessions limit
+1. In the **Utilization** panel, review the license count data.
 
-Sometimes it is useful to sign in to an account from multiple locations simultaneously. As of Grafana Enterprise 7.5+, accounts are limited to the number of concurrent sessions authorized in each license, which is typically three. A new session is created when a user signs in to Grafana from a new device, a different browser, or an incognito window. If a user signs in to Grafana from another tab or window within the same browser, then only one session is used.
+   - If you see **Admins & Editors** and **Viewers**, then you have a tiered license.
+   - If you only see **Users**, then you have a combined license.
 
-Given a limit of three sessions, the longest inactive session is signed out of Grafana when a fourth person signs in to the same account.
+## License restrictions common to both license types
 
-### Usage billing
+Your license is controlled by the following rules:
 
-You can request Grafana Labs to turn on usage billing to allow an unlimited number of active users. When usage billing is enabled, Grafana does not enforce active user limits or display warning banners. Instead, you are charged for active users above the limit, according to your customer contract.
+- **License expiration date:** The license includes an expiration date, which is the date when a license becomes inactive.
```

Comment:
```suggestion
**License expiration date:** The license includes an expiration date, which is the date when a license becomes inactive.
```

Reply:
For this change suggestion and the two that follow it, I think we are using two things here (an unordered and a definition list). Therefore, I removed one list type by eliminating `*`s.